R/col_types.R

```diff
@@ -308,16 +308,12 @@ colourise_cols <- function(cols, colourise = crayon::has_color()) {
     cols[[i]] <- switch(fname,
       col_skip = ,
       col_guess = cols[[i]],
-
```

Comment:
I think these blank lines are desirable. They seem to logically "chunk up" the different flavours of column (character-ish, logical, number-ish, datetime-ish). I'd keep this blank line and the ones right below (in the same call).